config-templates/redhat/odf-local/changelog.json

```diff
@@ -1,4 +1,21 @@
 [
+    {
+        "4.14": {
+            "status": "supported",
+            "latest-revision": {
+                "revision": "1",
+                "release-date": "30 January 2024",
```

Comment:
Considering the change in STAGE push plan to have on 29th Jan, please change release date to `31 January 2024`